internal/topology/variables/schema_test.go

```diff
@@ -86,8 +88,28 @@ func Test_convertToAPIExtensionsJSONSchemaProps(t *testing.T) {
 			}
 		})
 	}
-}
 
+	t.Run("pass for schema with default and enum", func(t *testing.T) {
```

Comment:
I added that unit test when chasing down why nullable variables are not defaulted.